app/views/component/locations/detail/_body.html.haml

```diff
@@ -286,4 +280,59 @@
   }
 
   initPrintButton();
+
```

Comment:
I see, We can leave it like this for now but we can work on having the full assets folders content to be precompiled. Also it would be good to use when possible jquery for consistency.